clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/CloudFoundryClientUtils.java

```diff
@@ -34,6 +34,8 @@
 import retrofit2.Response;
 
 public final class CloudFoundryClientUtils {
+  private static final String VALID_BINDING_NAME_PATTERN = "[^A-Za-z0-9-]+";
```

Comment:
Is there documentation we can point to that outlines the binding name pattern?

Reply:
It's not published as far as I can see, have to go digging in the CAPI code to find it.

As it turns out I missed that _ is allowed too, I'll open a PR to include that and link the relevant code tomorrow.